README.md

```diff
@@ -121,10 +121,205 @@ For example:
 ssh -L4000:localhost:4000 <MY_USERNAME>@<MY_HOSTNAME>
 ```
 
+### Set up access for AutoDBA Collector
+
+Set up the IAM role for the collector to access database metrics and logs.  Jump below to the section for your cloud provider (AWS or GCP).
+
+
+#### Amazon Web Services (AWS)
+
+##### Create IAM policy
+
+First, save this JSON to a file called **autodba_policy.json**
+
+```json
+{
+    "Version": "2012-10-17",
+        "Statement": [
+        {
+            "Action": [
+                "cloudwatch:GetMetricStatistics"
+            ],
+            "Effect": "Allow",
+            "Resource": "_"
+        },
+        {
+            "Action": [
+                "logs:GetLogEvents"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:logs:_:_:log-group:RDSOSMetrics:log-stream:_"
+        },
+        {
+            "Action": [
+                "rds:DescribeDBParameters"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:rds:_:_:pg:_"
+        },
+        {
+            "Action": [
+                "rds:DescribeDBInstances",
+            "rds:DownloadDBLogFilePortion",
+            "rds:DescribeDBLogFiles"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:rds:_:_:db:_"
+        },
+        {
+            "Action": [
+                "rds:DescribeDBClusters"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:rds:_:_:cluster:\*"
+        }
+    ]
+}
+```
+
+Then, run this command from the CLI:
+
+```bash
+aws iam create-policy
+--policy-name autodba
+--policy-document file://autodba_policy.json
+--description "Allow AutoDBA to access RDS"
+```
+
+##### Create IAM role
+
+First, run this command from the CLI:
+
+> **Tip:** replace `JSON_HERE` in the `aws` command below with the following JSON.
```

Comment:
Why not put it directly in there?

Otherwise, let's create a bash variable and assign this `JSON_HERE` to it. This way, users can copy-paste it.

Reply:
Fixed